test/CellDataTests/DiracDeltasTests.jl

```diff
@@ -37,6 +37,22 @@ degree = 2
 δ = DiracDelta{0}(model,tags=[2,4])
 @test sum(δ(v)) ≈ 5
 
+# Tests for DiracDelta at a generic Point in the domain #
+
+p = VectorValue(0.2,0.3)
```

Comment:
why do you use `VectorValue` and not `Point`?

Reply:
`Point` is not available for usage here, is has been defined in `Gridap.Fields` so just used `VectorValue` (as it seemed to me that the tests are also following the module hierarchy).

Reply:
I dont see any problem in `using Gridap.Fields` at the test. (perhaps I am missing something)

Reply:
No problem will use it now, I was in a misconception that even the tests follow the same module include hierarchy as the Gridap source code does.